spec/xcpretty/parser_spec.rb

```diff
@@ -444,6 +444,11 @@ module XCPretty
           @parser.parse(line)
         end
       end
+
+      it "parses ld warnings" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.